tests/unit/s2n_cert_info_test.c

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <errno.h>
+#include <fcntl.h>
+#include <openssl/pem.h>
+#include <openssl/x509.h>
+
+#include "api/s2n.h"
+#include "crypto/s2n_openssl_x509.h"
+#include "s2n_test.h"
+#include "testlib/s2n_testlib.h"
+
+int main(int argc, char **argv)
+{
+    BEGIN_TEST();
+
+    /* s2n_cert_get_cert_info */
```

Comment:
I'm having a little bit of a hard time following this test, and it's probably due to spacing/scoping? We sort of structure our test files like:
```
BEGIN_TEST();

// Define resources that are needed for all tests but aren't tests themselves

/* Comment explaining the first test */ 
{
}

/* Comment explaining the second test */
{
}

END_TEST();
```

Reply:
Sorry, I'm not quite following. My intention with this test was only to test a single function, `s2n_cert_get_cert_info`. Did you want me to break up the individual asserts with braces?
```       
        /* assert that cert info matches expected values */
        /* leaf and intermediate should have the same infos */
        /* root should be self-signed */
```

Reply:
Talked offline, the request was to move the test name down to before the for loop, instead of having it before the test cases.